packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -208,6 +211,9 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
         commands.registerCommand({ id: 'workbench.action.openSettings' }, {
             execute: () => commands.executeCommand(CommonCommands.OPEN_PREFERENCES.id)
         });
+        commands.registerCommand({ id: 'workbench.extensions.installExtension' }, {
+            execute: (id: string) => this.pluginServer.deploy(id)
```

Comment:
VS Code accepts either an URI (for installing a plain vsix file) or a plain string (for installing by ID), see https://github.com/microsoft/vscode/blob/33a49a9c6328a4576f1d8692bee70c267cbfe16b/src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts#L191